kitsune/users/views.py

```diff
@@ -328,6 +328,7 @@ def profile(request, template, username):
     groups = user_profile.user.groups.all()
     return render(request, template, {
         'profile': user_profile,
+        'country': user_profile.get_country_display(),
```

Comment:
You're still not using this variable, so you don't need to add it to the context.